### DISCUSSION TITLE
THX - do you plan the same for iOS

### DISCUSSION THREAD

**Discussion by I1eoThe0ne** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#147] [Ideas]
Hello and thanks a lot for this tool :)

do you plan to write an App for iOS as well ??
Is it relatively easy / similar to write an iOS app with the knowledge acquired in this project ???

Best regards

Thorsten

**Comment by vassudanagunta** [comment]
iOS is completely locked down. There is no way anyone will be able to do this on iOS unless it's jailbroken.

**Reply by I1eoThe0ne** [reply]
All right - Thx for pointing that out

**Reply by Jens-Oldenburg** [reply]
Had exactly the same question and as I can read, there is no hope. :-(
But thank you very much for this great tool on macOS!
